src/joint.dia.link.js

```diff
@@ -1773,6 +1795,18 @@ joint.dia.LinkView = joint.dia.CellView.extend({
                 this._z = null;
             }
 
+            var currentEnd = this.model.prop(arrowhead) || {};
+            var changed = initialEnd.id !== currentEnd.id || initialEnd.port !== currentEnd.port;
+
+            if (changed) {
+                if (initialEnd.id || initialEnd.port) {
+                    this.notify('link:disconnect', evt, arrowhead, this.paper.getModelById(initialEnd.id), this._initialMagnet);
```

Comment:
This should be `evt, arrowhead, elementView, magnetEl`. Not `evt, arrowhead, elementModel, magnetEl`.